src/components/Text/styles.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/variables.scss";
+
+@mixin HeadingText {
+  color: $color_white;
+  font-family: "Omnes";
+  font-weight: 700;
+  line-height: 125%;
+}
+
+@mixin BodyText {
+  color: $color_white;
+  font-family: "Operator Mono";
+  font-weight: 350;
```

Comment:
I feel like I typically see font-weights in 100, 200, 300, 400, etc, is 350 a valid weight? j curious idk if i've seen it before?

Reply:
Figma was showing the medium font weight as 350, could be a weird outlier 🤷🏽‍♂️